src/pages/index.tsx

```diff
@@ -1,12 +1,17 @@
 import { signIn, signOut, useSession } from "next-auth/react";
 import Head from "next/head";
 import Link from "next/link";
+import { useState, useEffect } from "react";
 
 import { api } from "~/utils/api";
 
-export default function Home() {
-  const hello = api.post.hello.useQuery({ text: "from tRPC" });
+interface Book {
+  id: number;
```

Comment:
i think we should get this from the backend types rather than defining it again on the frontend